include/odp/api/spec/ipsec.h

```diff
@@ -167,6 +167,13 @@ typedef struct odp_ipsec_inbound_config_t {
 	 */
 	odp_proto_chksums_t chksums;
 
+	/** Drop dummy packets (NH=59) instead of reporting it via
+	 *  @dummy_packet error bit.
+	 *
+	 *  @see dummy_packets_drop
+	 */
+	odp_bool_t dummy_packets_drop;
```

Comment:
Since you're referencing `tfc_padding_truncate`, the TFC prefix would seem consistent here. So `tfc_dummy_packets_drop` and comments should similarly reference TFC.